doc/hcr.rst

```diff
@@ -26,59 +26,61 @@ To install SDL2 you can use ``nimble install sdl2``.
 .. code-block:: nim
 
   # logic.nim
-  import sdl2/sdl
-
+  import sdl2
+  
```

Comment:
Can you cleanup the trailing whitespaces?

Reply:
I've cleaned up the whitespaces in commit `daccd8464b4186013ef5b59d13b3939512e2fdc1` resolved this